assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -183,32 +236,34 @@ function Carousel() {
             const isCurrentPage = index === currentPageIndex;
 
             return (
-              <DraggablePage
-                key={index}
-                onClick={handleClickPage(page)}
-                ariaLabel={
-                  isCurrentPage
-                    ? sprintf(
-                        __('Page %s (current page)', 'web-stories'),
-                        index + 1
-                      )
-                    : sprintf(__('Go to page %s', 'web-stories'), index + 1)
-                }
-                isActive={isCurrentPage}
-                pageIndex={index}
-                ref={(el) => {
-                  pageRefs.current[page.id] = el;
-                }}
-                width={PAGE_THUMB_WIDTH}
-                height={PAGE_THUMB_HEIGHT}
-              />
+              <Li key={index} marginRight={itemMarginRight} role="option">
```

Comment:
The `role="option"` should definitely not be here, as this is not an interactive element - clicking it has no meaning here nor can it receive focus, so it should not have an interactive role.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/411/commits/8cf49c41f56f500025310ee6e8258afd80005de6 anything beyond that will be handled by #10